src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -1,23 +1,39 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Set;
+import java.util.TreeSet;
+
 /**
- * <p>Реалізуйте метод `getUniqueCharacters(String fileName)` який приймає як параметр назву файлу.
- * Для цього використовуйте TreeSet. Файл містить букви латинського алфавіту і розділові знаки.</p>
- *
- * <p>Метод повинен відсортувати всі букви по алфавіту і повернути строку з 5 різних букв без
+ * Реалізуйте метод `getUniqueCharacters(String fileName)` який приймає як параметр назву файлу.
+ * Для цього використовуйте TreeSet. Файл містить букви латинського алфавіту і розділові знаки.
+ * Метод повинен відсортувати всі букви по алфавіту і повернути строку з 5 різних букв без
  * розділових знаків.
  * Якщо у файлі міститься менше п'яти різних букв, то потрібно повернути їх усі.
  * Можливий випадок, коли переданого файлу не існує, в такому разі потрібно викинути помилку
- * про відсутність файлу.</p>
- *
- * <p>Приклад 1: ur-BvT?^ ra w; p
- * Результат 1: abprt</p>
- *
- * <p>Приклад 2: A _f*c a?F
- * Результат 2: acf</p>
+ * про відсутність файлу.
+ * Приклад 1: ur-BvT?^ ra w; p
+ * Результат 1: abprt
+ * Приклад 2: A _f*c a?F
+ * Результат 2: acf
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int RESULT_LENGTH = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<String> characters = new TreeSet<>();
```

Comment:
Set<Character>. instead of String. Currently you are storing strings with length 1.

Reply:
Do you mean "Char instead of String"?  In this case, I won't be able to use String.join() method. In other words, String.join("", SOME SET OF CHARACTERS)  won't work.